File: Java/String-1/src/comboString/ComboString.java
package comboString;

public class ComboString {
    //Retorne uma String que seja do tipo shorter+longer+shorter, ou seja, a a maior String entre as duas deve vir no
    //meio da menor, ambas as String não podem ter o mesmo tamanho, mas podem ser nulas "".

    public String comboString(String a, String b) {
        int maior = Math.max(a.length(),b.length());
        if (a.isEmpty() && b.isEmpty()){
            return a+b+a;
        } else if (maior == a.length()){
            return b+a+b;
        } else {
            return a+b+a;
        }
    }
}

File: Python/String-1/combo_string.py
"""
Dada 2 Strings ("a" e "b"), retorne outra seguindo a lógica:
menor + maior + menor. As 2 strings não teram o mesmo tamanho, mas podem ser 0.
"""


def combo_string(a, b):
    menor = b if len(a) > len(b) else a
    maior = a if len(a) > len(b) else b
    return menor + maior + menor


print(combo_string("menos", "grande"))
